images/kor/README.md

```diff
@@ -0,0 +1,15 @@
+<!--monopod:start-->
+# kor
+| | |
+| - | - |
+| **OCI Reference** | `cgr.dev/chainguard/kor` |
+
+
+* [View Image in Chainguard Academy](https://edu.chainguard.dev/chainguard/chainguard-images/reference/kor/overview/)
+* [View Image Catalog](https://console.enforce.dev/images/catalog) for a full list of available tags.
+* [Contact Chainguard](https://www.chainguard.dev/chainguard-images) for enterprise support, SLAs, and access to older tags.*
+
+---
+<!--monopod:end-->
+
+A Golang Tool to discover unused Kubernetes Resources
```

Comment:
We'll need to fill out this documentation. This is the only thing our users would see when browsing the image.

Mainly some info as to how to run the image. Some examples:
- https://github.com/chainguard-images/images/tree/main/images/aws-ebs-csi-driver
- https://github.com/chainguard-images/images/tree/main/images/prometheus-statsd-exporter

The content may differ depending on the image, but high level:

- Summary of the image, link back to the upstream repository
- Information on how to use the image, i.e run via docker, note if appliciable back to the relevant upstream documentation which is appliciable
- example helm chart (not always applicable, but in this case we have one we use in tests so we should document how to use, and also link back to the upstream repo).

Reply:
+1. I personally find it valuable to have a sample input and output too.

Reply:
Thank-you @shivanshu1333 for actioning this one. Really like the updated README docs